epochStart/shardchain/trigger.go

```diff
@@ -509,17 +529,15 @@ func (t *trigger) checkIfTriggerCanBeActivated(hash string, metaHdr *block.MetaB
 		return false, 0
 	}
 
-	if metaHdr.IsStartOfEpochBlock() {
-		missingMiniblocksHashes, blockBody, err := t.peerMiniBlocksSyncer.SyncMiniBlocks(metaHdr)
-		if err != nil {
-			t.addMissingMiniblocks(metaHdr.Epoch, missingMiniblocksHashes)
-			log.Warn("processMetablock failed", "error", err)
-			return false, 0
-		}
-
-		t.epochStartNotifier.NotifyAllPrepare(metaHdr, blockBody)
+	missingMiniblocksHashes, blockBody, err := t.peerMiniBlocksSyncer.SyncMiniBlocks(metaHdr)
```

Comment:
so, we do not need this condition anymore?

Reply:
it is ensured that it is called only be start of epoch metablocks.

Reply:
ok